### PR TITLE
warn: add uw-longsd

### LINKED ISSUE #1897: warn: add {{Uw-longsd}}
Requested by Suntooooth at https://en.wikipedia.org/wiki/Wikipedia_talk:Twinkle#Adding_a_template

Add {{Uw-longsd}} as a single use notice

### PR DESCRIPTION
**Pull request by earlt612**

### CODE REVIEW

File: modules/twinklewarn.js [hunk 1]
```diff
@@ -1054,6 +1054,10 @@ Twinkle.warn.messages = {
 			label: 'Incorrect use of minor edits check box',
 			summary: 'Notice: Incorrect use of minor edits check box'
 		},
+		'uw-longsd': {
```
Comment (NovemLinguae) [inline modules/twinklewarn.js:1057]: Can we move this up a spot so that it's in alphabetical order?

File: modules/friendlytag.js [hunk 2]
```diff
@@ -835,6 +835,8 @@ Twinkle.tag.article.tagList = {
 			{ tag: 'BLP sources', description: 'BLP that needs additional sources for verification' },
 			{ tag: 'BLP unsourced', description: 'BLP that has no sources at all (use BLP PROD instead for new articles)' },
 			{ tag: 'More citations needed', description: 'needs additional references or sources for verification' },
+			{ tag: 'No signifigant coverage', description: 'article contains no sources with signifigant coverage of the subject' },
```
Comment (NovemLinguae) [inline modules/friendlytag.js:838]: This wasn't part of the original ticket. Are these two additions intentional?

File: modules/twinklewarn.js (hunk 1 above)
Comment (NovemLinguae) [inline modules/twinklewarn.js:1057]: Can we move this up a spot so that it's in alphabetical order?

**Review by NovemLinguae — CHANGES_REQUESTED** [review]

**Comment by earlt612** [comment]
Closed so that I can start with a fresh branch

**Pull request closed without merging**